backend/src/database/DatabaseSeeder.sql

```diff
@@ -0,0 +1,362 @@
+insert into Users (name, surname, email, password, role_id, username) values ('Harriot', 'Grahl', 'hgrahl1s@amazon.co.jp', 'b4Hbb|', 2, 'hgrahl1s');
+insert into Users (name, surname, email, password, role_id, username) values ('Filmore', 'Wiggett', 'fwiggett1t@tiny.cc', 'i92@leES', 2, 'fwiggett1t');
+insert into Users (name, surname, email, password, role_id, username) values ('Torey', 'Lanchbury', 'tlanchbury1u@sfgate.com', 'r5IF''`GI', 2, 'tlanchbury1u');
+insert into Users (name, surname, email, password, role_id, username) values ('Catlaina', 'Dalgetty', 'cdalgetty1v@utexas.edu', 'l3sLW', 2, 'cdalgetty1v');
+insert into Users (name, surname, email, password, role_id, username) values ('Carolyn', 'Pechan', 'cpechan1w@surveymonkey.com', 'm2=)Aj1', 2, 'cpechan1w');
+insert into Users (name, surname, email, password, role_id, username) values ('Aurel', 'Deerr', 'adeerr1x@huffingtonpost.com', 'q6oeV', 2, 'adeerr1x');
+
+
+INSERT INTO orders (user_id, order_status, total_price)
+VALUES
+(15, 'in-progress', 35.50),
+(16, 'delivered', 25.99),
+(17, 'canceled', 18.75),
+(18, 'in-progress', 40.00),
+(19, 'delivered', 28.60),
+(20, 'in-progress', 19.20),
+(21, 'delivered', 42.30),
+(22, 'canceled', 21.90),
+(23, 'in-progress', 30.00),
+(24, 'delivered', 35.50),
+(25, 'in-progress', 22.75),
+(26, 'delivered', 27.00),
+(27, 'in-progress', 23.90),
+(28, 'delivered', 29.99),
+(29, 'in-progress', 24.25),
+(30, 'canceled', 31.60),
+(31, 'in-progress', 18.30),
+(32, 'delivered', 22.60),
+(33, 'canceled', 38.00),
+(34, 'in-progress', 20.50),
+(35, 'delivered', 26.75),
+(36, 'in-progress', 33.00),
+(37, 'delivered', 40.40),
+(38, 'in-progress', 42.80),
+(39, 'canceled', 20.25),
+(40, 'delivered', 27.90),
+(41, 'in-progress', 19.10),
+(42, 'delivered', 41.00),
+(43, 'in-progress', 29.50),
+(44, 'delivered', 25.25),
+(45, 'in-progress', 23.40),
+(46, 'delivered', 39.70),
+(47, 'in-progress', 32.50),
+(48, 'canceled', 24.80),
+(49, 'delivered', 27.50),
+(50, 'in-progress', 34.00),
+(51, 'delivered', 28.99),
+(52, 'in-progress', 31.20),
+(53, 'delivered', 29.90),
+(54, 'canceled', 18.20),
+(55, 'in-progress', 43.99),
+(56, 'delivered', 39.50),
+(57, 'in-progress', 22.00),
+(58, 'delivered', 31.75),
+(59, 'in-progress', 41.00),
+(60, 'canceled', 18.50),
+(61, 'delivered', 34.90),
+(62, 'in-progress', 26.75),
+(63, 'delivered', 27.20),
+(64, 'in-progress', 32.40),
+(65, 'delivered', 23.10),
+(66, 'in-progress', 37.25),
+(67, 'delivered', 30.00),
+(68, 'in-progress', 22.50),
+(69, 'delivered', 34.40),
+(70, 'in-progress', 28.10),
+(71, 'delivered', 21.50),
+(72, 'canceled', 39.60),
+(73, 'in-progress', 25.30),
+(74, 'delivered', 22.75),
+(75, 'in-progress', 42.00),
+(76, 'delivered', 38.20),
+(77, 'canceled', 25.80),
+(78, 'in-progress', 19.99),
+(79, 'delivered', 24.40),
+(80, 'in-progress', 43.00);
+--ALTER SEQUENCE order_items_id_seq restart with 1;
```

Comment:
sta je ovo?

Reply:
Nista zanemari to je u komentaru, da mi resetira id na 1 kad pogrijesim insert, nebitno